src/purify.js

```diff
@@ -658,12 +658,15 @@ function createDOMPurify(window = getGlobal()) {
   /* Keep track of all possible SVG and MathML tags
    * so that we can perform the namespace checks
    * correctly. */
-  const ALL_SVG_TAGS = addToSet({}, TAGS.svg);
-  addToSet(ALL_SVG_TAGS, TAGS.svgFilters);
-  addToSet(ALL_SVG_TAGS, TAGS.svgDisallowed);
-
-  const ALL_MATHML_TAGS = addToSet({}, TAGS.mathMl);
-  addToSet(ALL_MATHML_TAGS, TAGS.mathMlDisallowed);
```

Comment:
Changed the `addToSet` function to be called only once, like `DEFAULT_ALLOWED_TAGS/DEFAULT_ALLOWED_ATTR`, rather than multiple times.

```js
const DEFAULT_ALLOWED_TAGS = addToSet({}, [
  ...TAGS.html,
  ...TAGS.svg,
  ...TAGS.svgFilters,
  ...TAGS.mathMl,
  ...TAGS.text,
]);

const DEFAULT_ALLOWED_ATTR = addToSet({}, [
  ...ATTRS.html,
  ...ATTRS.svg,   
  ...ATTRS.mathMl,
  ...ATTRS.xml,
]);
```